tests/5.0/teams/test_teams.F90

```diff
@@ -0,0 +1,73 @@
+!===--- test_teams.F90 -----------------------------------------------------===//
+!
+! OpenMP API Version 5.0 Nov 2018
+!
+! This test uses the teams directive on host and verifies the 
+! requested number of teams with requested number of threads were
+! created.
+!
+!//===----------------------------------------------------------------------===//
+
+#include "ompvv.F90"
+
+#define N 4
+#define K 2
+
+PROGRAM test_teams
+ 
+  USE iso_fortran_env
+  USE ompvv_lib
+  USE omp_lib
+  implicit none
+
+  OMPVV_TEST_OFFLOADING
+
+  OMPVV_TEST_VERBOSE(teams() .ne. 0)
+
+  OMPVV_REPORT_AND_RETURN()
+
+CONTAINS
+    INTEGER FUNCTION teams()
+    CHARACTER(len=300):: infoMsg
+    INTEGER,DIMENSION(OMPVV_NUM_TEAMS_DEVICE):: num_teams 
+    INTEGER,DIMENSION(OMPVV_NUM_THREADS_DEVICE):: num_threads
```

Comment:
This variable goes from lower bound `1` to `8` but below, the code has:
  `num_threads(omp_get_team_num())`
The problem is that `omp_get_team_num()` returns values from `0` to `omp_get_num_threads()-1`.

Hence, either the variable has to be defined as `DIMENSION( 0 : OMPVV_NUM_THREADS_DEVICE - 1)` or it has to be used as  `omp_get_team_num() + 1`.

Reply:
Thanks Tobias! Resolved this issue with most recent commit, decided to go with`omp_get_team_num() + 1`